doc/changes/changes_0.2.2.md

```diff
@@ -0,0 +1,11 @@
+# error-reporting-java 0.2.2, released 2020-11-30
```

Comment:
Update the date or use the latest RR for releasing (I think I sent you jar)

Reply:
But it will be an additional commit on main

Reply:
```suggestion
# error-reporting-java 0.2.2, released 2020-12-17
```